core/server/web/site/app.js

```diff
@@ -1,38 +1,40 @@
-var debug = require('ghost-ignition').debug('blog'),
-    path = require('path'),
-    express = require('express'),
-    setPrototypeOf = require('setprototypeof'),
-
-    // App requires
-    config = require('../../config'),
-    apps = require('../../services/apps'),
-    constants = require('../../lib/constants'),
-    storage = require('../../adapters/storage'),
-    urlService = require('../../services/url'),
-
-    // This should probably be an internal app
-    sitemapHandler = require('../../data/xml/sitemap/handler'),
-
-    // Route Service
-    siteRoutes = require('./routes'),
-
-    // Global/shared middleware
-    cacheControl = require('../middleware/cache-control'),
-    errorHandler = require('../middleware/error-handler'),
-    frontendClient = require('../middleware/frontend-client'),
-    maintenance = require('../middleware/maintenance'),
-    prettyURLs = require('../middleware/pretty-urls'),
-    urlRedirects = require('../middleware/url-redirects'),
-
-    // local middleware
-    servePublicFile = require('../middleware/serve-public-file'),
-    staticTheme = require('../middleware/static-theme'),
-    customRedirects = require('../middleware/custom-redirects'),
-    serveFavicon = require('../middleware/serve-favicon'),
-    adminRedirects = require('../middleware/admin-redirects'),
-
-    // middleware for themes
-    themeMiddleware = require('../../services/themes').middleware;
+const debug = require('ghost-ignition').debug('blog');
+const path = require('path');
+const express = require('express');
+
+// this module is missing in package.json
```

Comment:
?

Reply:
We are using `setprototypeof` in [2 places](https://github.com/TryGhost/Ghost/search?q=setprototypeof&unscoped_q=setprototypeof) (and it's present in our node_modules as it's a dependency of express) but it's not present explicitly anywhere in https://github.com/TryGhost/Ghost/blob/master/package.json

Reply:
Maybe we should consider switching to [native method](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Object/setPrototypeOf) instead as [they](https://www.npmjs.com/package/setprototypeof) [both](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Object/setPrototypeOf) describe to be doing the same thing :thinking:

Reply:
Very good catch, i thought this is a node require 👍